templates/zerver/help/editing-past-messages.md

```diff
@@ -0,0 +1,33 @@
+### Editing past messages
+
+ * If you want to edit a past message of your own after it has been
+   posted, hover your mouse over the message. You should see a
+   little pencil ![pencil](/static/images/help/pencil.png) at the end
+   of the message and a “down-chevron”
+   ![down chevron](/static/images/help/down_chevron.png) near the
+   upper right corner of the message. Clicking either one of these
+   will give you editing options.
+   
```

Comment:
Breaking this into bullet points will help visually clarify that there are two ways here, for example:

Zulip allows you to easily edit your messages after they have been posted. You can do this by hovering your mouse over the message and:
* Clicking on the pencil icon ![pencil](/static/images/help/pencil.png) at the end of the message
* Clicking on the down-chevron ![down chevron](/static/images/help/down_chevron.png) near the upper right corner of the message.

Clicking either one of those icons will give you editing options.